### DISCUSSION TITLE
Change Hotkey for Bandage/Runes

### DISCUSSION THREAD

**Discussion by eMalum** [HgAlexx/Buffet#24] [General]
Hey, first off thanks so much for all the work you've done.

Is there any way that I can change the modifier hotkey for Runes to be the same as Healthstones and make them only usable in combat? Currently the "Special" hotkey (default: CTRL) access both Bandages and Runes but I'd prefer if CTRL only handled Bandages, I'd like to change it so that Shift handles Healthstones and Runes (in combat only).

Thanks!

**Comment by HgAlexx** [comment]
Hello,

This could be achieved using custom macro.

Make a new custom macro and use the following code:

Change modifier to shift for rune mana macro:
```
local bests = ...

local drink = bests.conjuredDrink or bests.drink
local body = "#showtooltip\n/cast "

if bests.rune then
    body = body .. "[combat,mod:shift] item:" .. bests.rune.. "; "
end

if bests.manaPotion then
    body = body .. "[combat] item:" .. bests.manaPotion .. "; "
end

body = body .. "item:" .. (drink or "6948")

return body
```

I haven't tested it but it should be ok ;)


Let me know if you need a health macro too.

**Reply by eMalum** [reply]
That's perfect! 

Thank you so much.
